lab/util/pytorch.py

```diff
@@ -28,3 +33,52 @@ def get_modules(configs: Configs):
             modules[k] = value
 
     return modules
+
+
+class BaseDataset(Dataset):
+    pass
+
+
+class CustomDataset(BaseDataset):
```

Comment:
this may not be super general let me know ur thoughts

Reply:
Shall we move this out from `util/pytorch` and put in in `util/data/pytorch`?

Reply:
Why do you name it `CustomDataset`  shouldn't it be like `PandasDataset` or `CsvDataset`